### PR TITLE
Upgrade issue templates to the new issue template format

### LINKED ISSUE #4: Upgrade our issue templates to use GitHub issue forms ✍️
# Introduction

GitHub has recently rolled out a [public beta for their issue forms](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/) feature. This would allow you to create interactive issue templates and validate them 🤯.

Appwrite currently uses the older [issue template format](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates). Your task is to create **GitHub issue forms** for this repository. Please use [Appwrite's issue templates](https://github.com/appwrite/appwrite/tree/master/.github/ISSUE_TEMPLATE) as a reference for this PR.

### Tasks summary:

- [ ] Fork & clone this repository
- [ ] Prepare **bug report** issue form in `.github/ISSUE_TEMPLATE/bug.yaml`
- [ ] Prepare **documentation** issue form in `.github/ISSUE_TEMPLATE/documentation.yaml`
- [ ] Prepare **feature request** issue form in `.github/ISSUE_TEMPLATE/feature.yaml`
- [ ] Push changes to **master** and test issue forms on your fork
- [ ] Submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by MatteoGheza**
Closes #4
Issue examples:
- Feature request: https://github.com/MatteoGheza/sdk-for-ruby/issues/3
- Documentation update request: https://github.com/MatteoGheza/sdk-for-ruby/issues/4
- Bug report: https://github.com/MatteoGheza/sdk-for-ruby/issues/6

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@MatteoGheza Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

**Comment by MatteoGheza** [comment]
@lohanidamodar done.
BTW I found a typo in https://github.com/appwrite/appwrite/blob/master/.github/ISSUE_TEMPLATE/bug.yaml (line 61) so i opened a PR.
[](https://github.com/appwrite/appwrite/pull/1866)

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: textarea
+    id: code-to-reproduce
+    attributes:
+      label: "📜 Example code for reproducing this bug"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:21]: Lets keep it consistent across other repositories. This info can be included in the reproduction steps.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        - Version 0.8.x
+        - Version 0.7.x
+        - Version 0.6.x
+        - Different version (specify in environment)
+    validations:
+      required: true
+  - type: input
+    id: appwrite-sdk-version
+    attributes:
+      label: "🎲 Appwrite SDK for Ruby version"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:57]: Same here, let's keep it consistent at the moment.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
@MatteoGheza Great work. For now we want some few changes to make it consistent across repositories.
Thank you 👍🏻

**Comment by MatteoGheza** [comment]
Done.

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**